src/Licenses/LicenseFindFilterInterface.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses;
+
+use ArrowSphere\PublicApiClient\Licenses\Entities\License\ActiveSeats;
+use ArrowSphere\PublicApiClient\Licenses\Entities\License\License;
+use ArrowSphere\PublicApiClient\Licenses\Entities\License\Price;
+use ArrowSphere\PublicApiClient\Licenses\Entities\LicenseOfferFindResult;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\ActionFlags;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\Offer;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\ActionFlags as PriceBandActionFlags;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\Billing;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\PriceBand;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\Prices;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\SaleConstraints;
+
+/**
+ * Interface LicenseFindFilterInterface
+ *
+ * Description
+ */
+interface LicenseFindFilterInterface
```

Comment:
Sorry I know I told you to make an interface but in fact it would make a lot more sense as an enum.
Rename to ```LicenseFindFilterEnum```, or even better ```LicenseFindFieldEnum``` (because it's useful not only for filters but also for keywords and sort, so I think "field" is a better word)
Make it a class that extends ```AbstractEnum```
Create an Enum folder and put it in it